docs/system/system-metrics.md

```diff
@@ -916,6 +917,44 @@ This metric is [recommended][MetricRecommended].
 
 
 
+<!-- markdownlint-restore -->
+<!-- prettier-ignore-end -->
+<!-- END AUTOGENERATED TEXT -->
+<!-- endsemconv -->
+
+### Metric: `system.disk.limit`
+
+This metric is [opt-in][MetricOptIn].
+
+<!-- semconv metric.system.disk.limit(metric_table) -->
+<!-- NOTE: THIS TEXT IS AUTOGENERATED. DO NOT EDIT BY HAND. -->
+<!-- see templates/registry/markdown/snippet.md.j2 -->
+<!-- prettier-ignore-start -->
+<!-- markdownlint-capture -->
+<!-- markdownlint-disable -->
+
+| Name     | Instrument Type | Unit (UCUM) | Description    | Stability |
+| -------- | --------------- | ----------- | -------------- | --------- |
+| `system.disk.limit` | UpDownCounter | `By` | The total memory of the disk | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
What does "memory" mean here? Like how much memory the disk is using or how much space the disk has available for storage?

Reply:
Description changed in https://github.com/open-telemetry/semantic-conventions/pull/1356/commits/19271957bb5b1714badbdacb1f4f6e3f48b150f2